src/pages/ScreenRouting/playlistScreens/PlaylistGrid/PlaylistGrid.tsx

```diff
@@ -48,14 +43,7 @@ const PlaylistGrid: ScreenComponent<Playlist> = ({ data, isLoading }) => {
         {shouldShowFilter && <Filter name="genre" value={filter} defaultLabel="All" options={categories} setValue={setFilter} />}
       </header>
       <main className={styles.main}>
-        <CardGrid
-          playlist={filteredPlaylist}
-          onCardClick={onCardClick}
-          accessModel={accessModel}
-          isLoggedIn={!!user}
-          hasSubscription={!!subscription}
-          isLoading={isLoading}
-        />
+        <CardGrid playlist={filteredPlaylist} accessModel={accessModel} isLoggedIn={!!user} hasSubscription={!!subscription} isLoading={isLoading} />
```

Comment:
I usually suggest the layout with 1 property per line unless there are only 1-2 props.

Reply:
Looks like it's corrected by husky